light/client.go

```diff
@@ -268,16 +264,16 @@ func (c *Client) restoreTrustedLightBlock() error {
 	if err != nil {
 		return ErrGetTrustedBlockHeight{Err: err}
 	}
-
-	if lastHeight > 0 {
-		trustedBlock, err := c.trustedStore.LightBlock(lastHeight)
-		if err != nil {
-			return ErrGetTrustedBlock{Err: err}
-		}
-		c.latestTrustedBlock = trustedBlock
-		c.logger.Info("Restored trusted light block", "height", lastHeight)
+	if lastHeight == -1 {
```

Comment:
So, when the store is empty, `LastLightBlockHeight()` returns `-1, nil`.

Reply:
yes